quick_test.py

```
@@ -1,111 +1,118 @@
-        from zen.setup.mcp_setup import MCPSetupManager
+
+
         print("✅ MCPSetupManager imported")
-        
-        from zen.setup.troubleshooter import SetupTroubleshooter
+
+
         print("✅ SetupTroubleshooter imported")
-        
+
         return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)